tools/README.md

```diff
@@ -62,7 +62,7 @@ running this on save, but you can add a `git` hook to format before committing
 XSTYLER_PATH="dotnet tool run xstyler --"
 
 # Define path to XAML Styler configuration
-XSTYLER_CONFIG="../../XamlStyler.json"
+XSTYLER_CONFIG="XamlStyler.json"

```

Comment:
HUH. Is this relative to where you are when you run git???

You may need a `$(git rev-parse --show-toplevel)` (which will return the path to the repo root)

Reply:
I don't think so. I assumed it was relative to the file, but it seems like git is smart enough to always CWD to the root of the repo before running that file